script_runner/tasks.py

```diff
@@ -294,6 +286,10 @@ def eval_script(script_path, ctx, process=None):
 def download_resource(download_resource_func, script_path):
     split = script_path.split('://')
     schema = split[0]
+    suffix = script_path.split('/')[-1]
+    exec_tempdir = get_exec_tempdir() if get_exec_tempdir else None
+    target_path = tempfile.mktemp(suffix='-{0}'.format(suffix),
```

Comment:
It would be better to pass `target_path` only if get_exec_tempdir() is "enabled" as you want to preserve the default behavior of download_resource.